waku/waku_lightpush/protocol_metrics.nim

```diff
@@ -17,4 +17,5 @@ const
   peerNotFoundFailure* = "peer_not_found_failure"
   emptyRequestBodyFailure* = "empty_request_body_failure"
   emptyResponseBodyFailure* = "empty_response_body_failure"
+  emptyContentTopicFailure* = "empty_content_topic_failure"
```

Comment:
This would not be an "empty content topic failure" anymore, but a "decode rpc failure". Empty content topic is just one reason why RPC decoding would fail.